docs/clients/stopping_handler.rst

```diff
@@ -0,0 +1,121 @@
+Stopping Handler
+----------------
+A Vertiq module is considered stopped when it has been below its stopping speed continuously for some stopping time. 
+Anytime the module’s velocity goes above the stopping speed, it will reset the countdown on the stopping time.
+Vertiq modules execute a disarming process when disarming in order to try and safely come to a stop 
```

Comment:
You can ditch this stuff about the disarming process, its not stop specific really. Maybe just keep the first two sentences